jade/distribution.yaml

```diff
@@ -1036,6 +1036,23 @@ repositories:
       url: https://github.com/introlab/find_object_2d-release.git
       version: 0.5.1-0
     status: maintained
+  foo:
```

Comment:
Please choose a reasonable repository name for your entry, e.g. `naoqi_dcm_driver`.